regression/cbmc-library/memcpy-08/main.c

```diff
@@ -0,0 +1,22 @@
+#include <stdlib.h>
+#include <string.h>
+
+int __VERIFIER_nondet_int();
+
+void main()
+{
+  size_t size;
+  __CPROVER_assume(size >= 2);
+  // limit the size to ensure size * sizeof(int) wouldn't overflow, and we don't
+  // reach the maximum object size
+  __CPROVER_assume(size <= 1000);
```

Comment:
Good find, I couldn't figure out what exactly was causing the array comprehension to be emitted here. I had considered just writing a unit test for this particular case instead.